Prog/AddNew/Templates/New_projects_issue_119.sas

```diff
@@ -0,0 +1,31 @@
+/**************************************************************************
+ Program:  New_projects_issue_119.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Will Oliver
+ Created:  9/10/18
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Add new projects to Preservation Catalog. 
+ 
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( MAR, local=n )
+%DCData_lib( RealProp, local=n )
+%DCData_lib( ROD, local=n )
+%DCData_lib( DHCD, local=n )
+
+
+%Add_new_projects(
+  input_file_pre = New_projects_issue_119
```

Comment:
Add the value `S8-MR` to the Program column in **L:\Libraries\PresCat\Raw\AddNew\New_projects_issue_119_subsidy.csv**.